docs/data/data-grid/components/components.md

```diff
@@ -56,13 +56,13 @@ Pass the `showToolbar` prop to the `<DataGrid />` component to enable the defaul
 
 You can also compose your own toolbar using the [Toolbar component](/x/react-data-grid/components/toolbar/).
 
-#### Legacy toolbar components
+### Legacy Toolbar
```

Comment:
Important, we want to see it on the TOC and it's an alternative to `### Toolbar` so same heading level.